org.eclipse.scout.rt.client/src/main/java/org/eclipse/scout/rt/client/ui/basic/table/TableEventBuffer.java

```diff
@@ -596,6 +607,13 @@ protected <TYPE> void mergeCollections(Collection<TYPE> source, List<TYPE> targe
       source.removeIf(sourceElement -> !targetSet.add(sourceElement));
       target.addAll(0, source);
     }
+
+    /**
+     * Merge maps, such that, if an element is in both maps, only the one of the second map (later event) is kept.
```

Comment:
Is this correct? Only the keys seem to be merged, the values are being overwritten. If there are two different events for the same row, but with different updated columns, shouldn't both columns be in the resulting map afterwards? 

Add test case for this in unit test.

Reply:
Fixed and added test